local_classes.py

```diff
@@ -24,20 +24,7 @@ class Day:
     4. Saving the data strusture of the weaving object to the directory
     """
 
-    days_per_month = {
```

Comment:
Dict takes much more memery. So, as fas as keys are integer values, I have to use basic list